README.md

```diff
@@ -1 +1,34 @@
 # kube-linter
+
+kube-linter is a static analysis tool that checks Kubernetes YAML files to ensure the applications represented in them
+adhere to best practices.
+
+In detail, `kube-linter` is a binary that takes in paths to YAML files, and runs a list of checks
+against them. If any lint errors are found, they are printed to standard error, and `kube-linter` returns a non-zero 
+exit code.
+
+The list of checks that is run is configurable. `kube-linter` comes with several built-in checks, only some of which
+are enabled by default. Users can also create custom checks.
+
+## Install
+
+If you have `go` installed, you can run `go get golang.stackrox.io/kube-linter/cmd/kube-linter`.
+
+`kube-linter` binaries can be downloaded from [the Releases page](https://github.com/stackrox/kube-linter/releases).
+Download the `kube-linter` binary, and add it to your PATH.
```

Comment:
IMHO a readme in an open source repository should always include custom build instructions and contributing guidelines (or references to these)

Reply:
👍

Reply:
Added instructions for building from source. Contributing guidelines, will do in a follow-up -- we still need to iron those out.